.github/workflows/pythonapp.yml

```
@@ -0,0 +1,91 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Exercises check
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

It's currently only configured to run on `master`. Try this:
```suggestion
on:
  push:
    branches: [ master ]
  pull_request:
```